<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Costs</class>
 <widget class="QMainWindow" name="Costs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(170, 255, 255);
	border-color: rgb(0, 0, 127);
border-radius: 12px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 14px;
}</string>
    </property>
    <property name="text">
     <string>           Витрати</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 255);
	border-color: rgb(0, 0, 127);
border-radius: 8px;
}
</string>
    </property>
    <property name="title">
     <string>Калькулятор</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
background-color: rgb(255, 170, 255);
border-color: rgb(0, 0, 127);
color: rgb(0, 0, 127);
border-radius: 6px;
}</string>
     </property>
     <property name="text">
      <string> Калькулятор вигідності</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_result">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>280</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(255, 85, 0);
	color: rgb(255, 255, 0);
	border-color: rgb(255, 255, 0);
border-radius: 8px;
}
QPushButton:hover:!pressed { 
background-color: white;
border-color: black;
color: black;
}</string>
     </property>
     <property name="text">
      <string>Обрахувати</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>291</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
border-radius: 4px;
	color: rgb(255, 0, 0);
	border-color: rgb(255, 0, 0);
	background-color: rgb(170, 255, 127);
padding: 8px 20px;
}</string>
        </property>
        <property name="text">
         <string>Вік:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_age">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
	border-color: rgb(0, 0, 0);
	color: rgb(0, 0, 0);
border-radius: 4px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
border-radius: 4px;
	color: rgb(255, 0, 0);
	border-color: rgb(255, 0, 0);
	background-color: rgb(170, 255, 127);
}</string>
        </property>
        <property name="text">
         <string>     Чемпіонат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
border-radius: 4px;
	color: rgb(255, 0, 0);
	border-color: rgb(255, 0, 0);
	background-color: rgb(170, 255, 127);
}</string>
        </property>
        <property name="text">
         <string>       Позиція</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>89</y>
       <width>141</width>
       <height>248</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_epl">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 20px;
}</string>
        </property>
        <property name="text">
         <string>EPL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_laliga">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 20px;
}</string>
        </property>
        <property name="text">
         <string>LaLiga</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_seriaa">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 20px;
}</string>
        </property>
        <property name="text">
         <string>Seria A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_bundesliga">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 20px;
}</string>
        </property>
        <property name="text">
         <string>Bundesliga</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_liga1">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 20px;
}</string>
        </property>
        <property name="text">
         <string>Liga 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_another">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 20px;
}</string>
        </property>
        <property name="text">
         <string>Інша ліга</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>91</y>
       <width>141</width>
       <height>164</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_frw">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 15px;
}</string>
        </property>
        <property name="text">
         <string>Нападаючий</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_mdf">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 15px;
}</string>
        </property>
        <property name="text">
         <string>Півзахисник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_def">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 15px;
}</string>
        </property>
        <property name="text">
         <string>Захисник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_gk">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 127);
	border-color: rgb(170, 0, 0);
border-radius: 4px;
padding: 10px 15px;
}</string>
        </property>
        <property name="text">
         <string>Воротар</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
border-radius: 4px;
	color: rgb(255, 0, 0);
	border-color: rgb(255, 0, 0);
	background-color: rgb(170, 255, 127);
padding: 5px 5px;
}</string>
        </property>
        <property name="text">
         <string>Трансферна вартість (од.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_price">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
	border-color: rgb(0, 0, 0);
	color: rgb(0, 0, 0);
border-radius: 4px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(255, 255, 0);
	border-color: rgb(255, 170, 0);
	background-color: rgb(255, 0, 0);
border-radius: 10px;
}
QPushButton:hover:!pressed { 
background-color: white;
border-color: black;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
color: rgb(170, 0, 0);
	background-color: rgb(255, 255, 255);
	border-color: rgb(0, 0, 0);
border-radius: 6px;
}</string>
    </property>
    <property name="title">
     <string>Клубні витрати</string>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: rgb(255, 0, 0);
	background-color: rgb(255, 255, 0);
	border-color: rgb(255, 0, 0);
border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string>Усі суми вводити у одиницях!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(255, 0, 0);
	background-color: rgb(85, 255, 0);
	border-color: rgb(255, 0, 0);
border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string>Зарплата гравців:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(255, 0, 0);
	background-color: rgb(85, 255, 0);
	border-color: rgb(255, 0, 0);
border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string>Зарплата персоналу:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(255, 0, 0);
	background-color: rgb(85, 255, 0);
	border-color: rgb(255, 0, 0);
border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string>Витрати на стадіон:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(255, 0, 0);
	background-color: rgb(85, 255, 0);
	border-color: rgb(255, 0, 0);
border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string>Витрати на трансфери:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(255, 0, 0);
	background-color: rgb(85, 255, 0);
	border-color: rgb(255, 0, 0);
border-radius: 6px;
}</string>
        </property>
        <property name="text">
         <string>Витрати на транспорт:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>15</y>
       <width>231</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_salaryplayers">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 144);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 170, 0);
border-radius: 4px;
padding: 5px 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_salarypersonal">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 144);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 170, 0);
border-radius: 4px;
padding: 5px 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_stadiumprice">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 144);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 170, 0);
border-radius: 4px;
padding: 5px 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_transfers">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 144);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 170, 0);
border-radius: 4px;
padding: 5px 2px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_transport">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
background-color: rgb(255, 255, 144);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 170, 0);
border-radius: 4px;
padding: 5px 2px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>265</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
background-color: rgb(255, 170, 127);
	border-color: rgb(255, 170, 0);
	color: rgb(170, 0, 0);
border-radius: 6px;
width: 25px;
}</string>
        </property>
        <property name="text">
         <string>Загалом:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_resultat">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	background-color: rgb(170, 255, 127);
	color: rgb(255, 0, 0);
	border-color: rgb(255, 170, 0);
border-radius: 4px;
width: 300px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_res">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>392</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(255, 85, 0);
	color: rgb(255, 255, 0);
	border-color: rgb(255, 255, 0);
border-radius: 8px;
}
QPushButton:hover:!pressed { 
background-color: white;
border-color: black;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Розрахувати</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
